src/less/type-utilities.less

```diff
@@ -0,0 +1,81 @@
+
+@import (reference) "./fonts.less";
+
+/* Basscss Typography */
+
+.font-family-inherit { font-family: inherit }
+
+.font-size-inherit { font-size: inherit }
+
+.text--decoration-none { text-decoration: none }
```

Comment:
What is the philosophy around making these as explicit classes vs. just including these styles as part of larger classes?

eg.
```
.my-link {
  color: #000;
}
...
<a class="my-link text--decoration-none" href="#">Click here</a>
```

vs.

```
.my-link {
  color: #000;
  text-decoration: none;
}
...
<a class="my-link" href="#">Click here</a>
```